testing_and_setup/compass/ocean/drying_slope/meshes/transect/config_driver.xml

```diff
@@ -0,0 +1,5 @@
+<driver_script name="run_test.py">
+	<case name="build_transect_mesh">
+		<step executable="./run.py" quiet="true" pre_message=" * Running build_transect_mesh" post_message="     Complete"/>
+	</case>
+</driver_script>
```

Comment:
@caozd999, should the name of the directory for this step be `idealized_transect` instead of `transect` so that it matches with the corresponding forward case?

Reply:
ok, I will change it accordingly. Thanks!

Reply:
Oh I see, that makes sense.

Reply:
Feel free to change it back if you want

Reply:
Never mind, this looks good too -- it's now easier to link them.